src/modules/Security.rb

```diff
@@ -547,6 +563,25 @@ def write_shadow_config
       shadow_config.save
     end
 
+    # Set SELinux settings
+    #
+    # @return true on success
+    def write_selinux
+      return if @Settings["SELINUX_MODE"].empty?
+
+      selinux_config.mode = @Settings["SELINUX_MODE"]
+
+      # Y2Security::Selinux.mode= fallbacks to the "Disabled" mode when wrong value is given.
+      # In consequence, let's check that the mode still being the same before saving it.
+      if selinux_config.mode.id.to_s != @Settings["SELINUX_MODE"]
+        log.info("The set SELinux mode does not match with the requested one. Not saving it.")
+        false
+      else
```

Comment:
As stated there, the `Y2Security::Selinux.mode = :wrong_mode` [will fallback to the disabled mode](https://github.com/yast/yast-security/blob/7c23e55f7af44eae59691c91ef5a65c70c0b497e/src/lib/y2security/selinux.rb#L164-L167). Which most probably is wrong (at least I see it wrong now) and it should be changed to simply do not touch the mode. 

@jreidinger, friendly ping :point_up:

Anyway, probably this extra check (added by me while updating #85) makes no sense at all and we must trust in whatever the _combination_ of  selinux_config.mode= + selinux_config.save does.

Reply:
well, in this situation when mode is unknown I would use common AutoinstIssues so it report wrong value and list of possible ones. As it is profile error and use later disabled in that case.

Reply:
> well, in this situation when mode is unknown I would use common AutoinstIssues

If I'm not wrong, this is already done at https://github.com/yast/yast-security/pull/88/commits/222a7a530115870a25c7554ed0069bb22d044b99

So, could we remove this ugly extra check from here safely?